core/modules/utils/dom/modal.js

```diff
@@ -26,8 +26,12 @@ Display a modal dialogue
 Options include:
 	downloadLink: Text of a big download link to include
 */
-Modal.prototype.display = function(title,options) {
+Modal.prototype.display = function(event,options) {
```

Comment:
We shouldn't change the type of an existing parameter. Perhaps we can use `options.event` instead?

Reply:
@Jermolene , done!